code/game/gamemodes/cult/runes.dm

```diff
@@ -168,7 +172,7 @@
 	else if(user.loc == get_turf(src))
 		speak_incantation(user, "Sas[pick("'","`")]so c'arta forbici!")
 		if(do_after(user, 4 SECONDS, bonus_percentage = 25))
-			user.visible_message(SPAN_WARNING("\The [user] disappears in a flash of red light!"), SPAN_WARNING("You feel as your body gets dragged into the dimension of Nar-Sie!"), "You hear a sickening crunch.")
+			user.visible_message(SPAN_WARNING("\The [user] disappears in a flash of red light!"), SPAN_WARNING("You feel as your body gets dragged into the dimension the Scarlet King!"), "You hear a sickening crunch.")
```

Comment:
See if you can replace some of these `visible_message`s and `to_chat`s with balloon alerts. This goes for all of your changes.

Reply:
no